src/Components/Theme/theme.ts

```diff
@@ -80,64 +94,44 @@ const ALLOWED_STYLES = [
+	'favorites',
+	'grid',
+	'contextmenu',
+	'prompt',
+	'preview',
+	'properties',
+];
+
 /**
  * Change page theme
  * @param {string} theme - The current theme
  * @returns {Promise<void>}
  */
-const changeTheme = async (theme?: string, category?: '*' | 'root' | 'tabbing' | 'favorites' | 'grid'): Promise<void> => {
+const changeTheme = async (theme?: string, category?: Category): Promise<void> => {
 	if (!category) category = '*';
-	const appearance = await Storage.get('appearance');
+	const appearance = (await Storage.get('appearance')) || {};
 	if (category === '*' || category === 'root') {
-		document.body.style.setProperty('--edge-radius', appearance?.frameStyle === 'os' || !isTauri ? '0px' : '10px');
-		document.body.style.fontSize = appearance?.fontSize ?? '16px';
-		document.documentElement.style.fontSize = appearance?.fontSize ?? '16px';
-		document.body.style.fontFamily = appearance?.fontFamily ?? 'system-ui';
-		document.documentElement.style.fontFamily = appearance?.fontFamily ?? 'system-ui';
-		if (isTauri) {
-			document.body.style.setProperty(
-				'--sidebar-transparency',
-				appearance?.transparentSidebar ?? true ? appearance?.windowTransparency ?? '0.8' : '1'
-			);
-			document.body.style.setProperty(
-				'--workspace-transparency',
-				appearance?.transparentWorkspace ?? false ? appearance?.windowTransparency ?? '0.8' : '1'
-			);
-			document.body.style.setProperty(
-				'--topbar-transparency',
-				appearance?.transparentTopbar ?? false ? appearance?.windowTransparency ?? '0.8' : '1'
-			);
-		} else {
-			document.body.style.setProperty('--sidebar-transparency', '1');
-			document.body.style.setProperty('--workspace-transparency', '1');
-			document.body.style.setProperty('--topbar-transparency', '1');
-		}
-
-		document.querySelectorAll<HTMLElement>('.sidebar-hover-effect').forEach((obj) => {
-			obj.style.borderRadius = '6px';
-			changeElementTheme(obj, 'sidebarBackground', 'background', theme);
-			if (obj.getAttribute('being-listened') !== 'true') {
-				obj.setAttribute('being-listened', 'true');
-				obj.addEventListener('mousemove', (e) => {
-					const rect = (e.target as Element).getBoundingClientRect();
-					const x = e.clientX - rect.left;
-					const y = e.clientY - rect.top;
-					const elementIsActive = obj.classList.contains('active');
-					if (elementIsActive) obj.onmouseleave = null;
-					else {
-						obj.style.background = `radial-gradient(circle at ${x}px ${y}px, ${getElementStyle('animation.sidebar', currentTheme)} )`;
-						obj.onmouseleave = () => {
-							obj.style.background = null;
-							obj.style.borderImage = null;
-						};
-					}
-				});
-			}
-		});
+		const root = document.documentElement;
+		const opacity = appearance.windowTransparency || '0.8';
+		root.style.fontSize = appearance.fontSize || '16px';
+		root.style.fontFamily = appearance.fontFamily || 'system-ui';
+		root.style.setProperty('--edge-radius', appearance.frameStyle === 'os' || !isTauri ? '0px' : '10px');
```

Comment:
@kimlimjustin I move it to `root` instead. All the docs I've read before use `root` instead of `body`. I just use `root` for consistency, and I haven't encountered any missing `border-radius`

Reply:
oh okay, thanks!